fennel/client/client.py

```diff
@@ -301,6 +301,22 @@ def extract_historical_features(
         else:
             return pd.Series(response.json())
 
+    def lookup(
+        self, dataset_name: str, keys: List[Dict[str, Any]], fields: List[str]
+    ) -> tuple[Union[pd.DataFrame, pd.Series], pd.Series]:
+        req = {
+            "keys": keys,
+            "fields": fields,
+        }
```

Comment:
This api only looks up latest ? why not also have asof semantics?

Reply:
That's a good idea. The current rest server doesn't have the asof supported, probably because it's not there yet when we started the metadata API project.